src/frontend/Felles/Linker/AInntekt/AInntektLink.ts

```diff
@@ -0,0 +1,21 @@
+import { apiLoggFeil, preferredAxios } from '../../../App/api/axios';
+import { AxiosError, AxiosResponse } from 'axios';
+import { AppEnv } from '../../../App/api/env';
+
+const errorMelding = 'Noe feilet vid henting av link mot a-inntekt';
+
+export const lagAInntektLink = async (appEnv: AppEnv, personIdent: string): Promise<string> => {
+    return await preferredAxios
```

Comment:
Trengs det et kall for å redirecte til a-inntekt? 😱

Reply:
Ja i context ac personIdent, den lager en redirect-link som gjør att man kommer direkt til personen sin a-inntekt

Reply:
Spennende 🤔